definitions/tools/star_fusion_detect.wdl

```diff
@@ -31,42 +30,32 @@ task starFusionDetect {
     mkdir ~{genome_lib_dir} && unzip -qq ~{star_fusion_genome_dir_zip} -d ~{genome_lib_dir}
     /usr/local/src/STAR-Fusion/STAR-Fusion --CPU ~{cores} \
         --genome_lib_dir ~{genome_lib_dir} \
-        -J ~{junction_file} --output_dir ~{fusion_output_dir} --STAR_PATH ~{star_path} \
+        --output_dir ~{fusion_output_dir} --STAR_PATH ~{star_path} \
         ~{true="--examine_coding_effect" false="" examine_coding_effect} \
         ~{if defined(fusioninspector_mode) then "--FusionInspector " + fusioninspector_mode else ""} \
+        --STAR_outSAMattrRGline "~{sep=" , " outsam_attrrg_line}" \
         --left_fq ~{sep="," fastq} --right_fq ~{sep="," fastq2}
   >>>
 
   output {
+    # star fusion outputs 
     File fusion_predictions = fusion_output_dir + "/star-fusion.fusion_predictions.tsv"
     File fusion_abridged = fusion_output_dir + "/star-fusion.fusion_predictions.abridged.tsv"
     File? coding_region_effects = fusion_output_dir + "/star-fusion.fusion_predictions.abridged.coding_effect.tsv"
+    # Fusion inspector outputs
     # if no mode specified, this will just not find any files
     Array[File] fusioninspector_evidence = glob(fusion_output_dir + "/FusionInspector-" + select_first([fusioninspector_mode, ""]) + "/finspector.*")
+    File fusioninspector_log = fusion_output_dir + "/FusionInspector.log"
+    # STAR alignment files
+    File aligned_bam = fusion_output_dir + "/Aligned.out.bam"
+    File log_final = fusion_output_dir + "/Log.final.out"
+    File log = fusion_output_dir + "/Log.out"
+    File log_progress = fusion_output_dir + "/Log.progress.out"
+    File splice_junction_out = fusion_output_dir + "/SJ.out.tab"
+    File chim_junc = fusion_output_dir + "/Chimeric.out.junction"
+    # STAR also outputs gene counts file just like Kallisto
+    File gene_counts = fusion_output_dir + "/ReadsPerGene.out.tab"
   }
 }
 
-workflow wf {
-  input {
-    File star_fusion_genome_dir_zip
-    File junction_file
-    String? fusion_output_dir
-    String? star_path
-    Boolean? examine_coding_effect
-    String fusioninspector_mode  # enum [inspect, validate]
-    Array[File] fastq
-    Array[File] fastq2
-  }
-
-  call starFusionDetect {
-    input:
-    star_fusion_genome_dir_zip=star_fusion_genome_dir_zip,
-    junction_file=junction_file,
-    fusion_output_dir=fusion_output_dir,
-    star_path=star_path,
-    examine_coding_effect=examine_coding_effect,
-    fusioninspector_mode=fusioninspector_mode,
-    fastq=fastq,
-    fastq2=fastq2
-  }
-}
+workflow wf { call starFusionDetect { input: } }
```

Comment:
What are the benefits of blanking out this workflow?

Reply:
None that i can think of, except IIRC its to avoid repetition in case you have a lot in input variables just like [here](https://github.com/griffithlab/analysis-wdls/blob/main/definitions/tools/star_align_fusion.wdl#L88)